packages/core/bootstrap/src/lib/provider-limits/limits.json

```diff
@@ -1027,5 +1027,15 @@
   "xbto": {
     "http": {},
     "ws": {}
+  },
+  "wootrade": {
+    "http": {},
+    "ws": {     
+      "private": {
+        "connections": 1,
+        "subscriptions": 1,
+        "note": "you need to request the application_id directly with wootrade"
```

Comment:
Are these limits actually 1, or are there no limits?
Can we do -1 if there are no limits, and add a note that docs doesn't mention limits?

Reply:
remove :) there is not information about ws limits